pkg/mutation/stats_reporter.go

```diff
@@ -0,0 +1,94 @@
+package mutation
+
+import (
+	"context"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/metrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	mutationSystemIterationsMetricName = "mutation_system_iterations"
+)
+
+// SystemConvergenceStatus defines the outcomes of the attempted mutation of an object by the
+// mutation System.  The System is meant to converge on a fully mutated object.
+type SystemConvergenceStatus string
+
+var (
+	systemConvergenceKey = tag.MustNewKey("success")
+
+	// JULIAN need to try making this a boolean somehow.
+
+	// SystemConvergenceTrue denotes a successfully converged mutation system request.
+	SystemConvergenceTrue SystemConvergenceStatus = "true"
+	// SystemConvergenceFalse denotes an unsuccessfully converged mutation system request.
+	SystemConvergenceFalse SystemConvergenceStatus = "false"
+
+	systemIterationsM = stats.Int64(
+		mutationSystemIterationsMetricName,
+		"The distribution of Mutator ingestion durations",
+		stats.UnitDimensionless)
+)
+
+func init() {
+	if err := register(); err != nil {
```

Comment:
Ditto comment above - move contents of register() to init if it should only be called on package initialization.

Reply:
Done